app/src/pages/inside/manualLaunchesPage/launchSidePanel/launchSidePanel.scss

```
@@ -62,17 +62,73 @@
+        opacity: 1;
+      }
+    }
+  }
+
+  .hash-icon {
+    flex-shrink: 0;
+    display: block;
+    width: 17px;
+    height: 16px;
+    color: var(--Text-Secondary, var(--rp-ui-base-e-400));
+    fill: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)